implementation/src/main/java/io/smallrye/graphql/bootstrap/Classes.java

```diff
@@ -38,59 +45,69 @@
  * @author Phillip Kruger (phillip.kruger@redhat.com)
  */
 public class Classes {
+    private static final Map<String, Class> loadedClasses = new HashMap<>();
 
     private Classes() {
     }
 
+    public static Class<?> loadClass(String className) {
+
+        if (isPrimative(className)) {
```

Comment:
I think this is a type: should be 'primitive'

Reply:
Thanks :) Fixing